query/src/api/rpc/exchange/exchange_channel.rs

```diff
@@ -0,0 +1,105 @@
+use std::ops::Deref;
+use std::sync::atomic::AtomicUsize;
+use std::sync::atomic::Ordering;
+use std::sync::atomic::Ordering::Acquire;
+use std::sync::Arc;
+
+use async_channel::Sender;
+use common_arrow::arrow_format::flight::data::FlightData;
+use common_exception::Result;
+
+use crate::api::rpc::packet::DataPacket;
+
+// Different from async_channel::Sender
+// It is allowed to close the channel when has one reference.
+// In other words, we only record the number of cloned.
+pub struct FragmentSender {
```

Comment:
Does the term `Fragment` means a **data chunk** to transport?

Reply:
It's Query Fragment sender.